### ISSUE TITLE
Wizdom API Documentation

### ISSUE THREAD

**Issue by TwilightMercy** [wizupdate/service.subtitles.wizdom#6]
Hi,

Do you have Wizdom API's documentation page?
Thanks!

**Comment by razsteinmetz** [comment]
Hi,
Here is what I got from them a while back:

This is what we have for now.
http://json.wizdom.xyz/[IMDB_ID].json
http://zip.wizdom.xyz/[SUB_ID].zip
http://posters.wizdom.xyz/[IMDB_ID].jpg 

http://json.wizdom.xyz/search.php?search=[QUERY]
http://json.wizdom.xyz/search.php?action=by_id&imdb=[IMDB_ID]
http://json.wizdom.xyz/search.php?action=guessit&filename=[FILE_NAME]

enjoy.

**Comment by wizupdate** [comment]
Raz is correct

**Comment by TwilightMercy** [comment]
Thank you for the reply! But these URLs doesn't work at all @razsteinmetz @wizupdate 

From what I know, the current Wizdom's API URL is:
`http://wizdom.xyz/api/search`

Example usage of searching by IMDb ID:
`http://wizdom.xyz/api/search?action=by_id&imdb=tt10640346&season=0&episode=0`

I was looking for more documentation for currently working API, if exists

**Comment by razsteinmetz** [comment]
You are right, those were old links.  I wrote a golang library recently (not yet out on github) and here is what I figured:
(none are documented)

## look for a specific subtitle imdb id (with possible season/episode filtering)
GET wizdom.xyz/api/search?action=by_id&imdb=tt1442462
GET wizdom.xyz/api/search?action=by_id&imdb=tt1442462&season=2&episode=3
GET wizdom.xyz/api/files/sub/id  (id is the id of the subtitle!)

## search by title of tv/movie
GET https://wizdom.xyz/api/search?search=hangman

## get latest releases (used on the website to show the titles)
GET https://wizdom.xyz/api/releases?type=tv&order=new&page=0
GET https://wizdom.xyz/api/releases?type=movie&order=new&page=0
GET https://wizdom.xyz/api/releases/tt1027718  [return all data on an IMDBID!]


hope this helps.
Raz

**Comment by TwilightMercy** [comment]
Thank you very much.
Do you know anything about `version` (release title) parameter? for `action=by_id` search
Example:
`http://wizdom.xyz/api/search?action=by_id&imdb=tt10640346&season=0&episode=0&version=Babylon`

Are there more possible actions beside `by_id`? (imdb id)

**Comment by razsteinmetz** [comment]
That is all i found out.
